Rakefile

```diff
@@ -1,10 +1,23 @@
-task :default => [ :"tests:ci" ]
+require "tmpdir"
 
-desc "Build the website"
-task :build do
-  sh "bundle exec jekyll build"
+# Clones the current git repository into a temp directory that shares the
+# object store. Also sets up a github remote.
+#
+# Commands can be run against this clone can be used to manipulate the
+# github remote without affecting your current checkout.
+def run_in_cloned_git
+  Dir.mktmpdir("vets-website-deploy") do |dir|
+    sh "git clone -l --single-branch -s . #{dir}"
+    sh "cd #{dir}"
+    sh "git remote | grep github && git remote remove github"
```

Comment:
Nice catch! Ugh. I'll need to rethink that tomorrow when I have more brain juice.